### PR TITLE
attempt to make the failed-to-load-entity-script error-message be more helpful

### PR DESCRIPTION
**Pull request by sethalves**
- attempt to make the failed-to-load-entity-script error-message be more helpful

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- open log
- filter log with "error"
- load/reload http://headache.hungry.com/~seth/hifi/es-test/entitySpawner.js
- look in log for helpful error message about myEntityScript.js failing to load

**Comment by hifi-gustavo** [comment]
Build console output for commit: 21c2db7cdce8ffe32658b515790999d5ba56db99 available at: https://gist.github.com/4ffafd84154a88eb3ccca2b3128c7378<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7600.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7600.dmg

**Comment by sethalves** [comment]
https://app.asana.com/0/26485730907942/108656098018520
https://app.asana.com/0/26485730907942/95580254898057

File: libraries/script-engine/src/ScriptEngine.cpp [hunk 1]
```diff
@@ -1089,9 +1089,18 @@ void ScriptEngine::entityScriptContentAvailable(const EntityItemID& entityID, co
...
     if (!testConstructor.isFunction()) {
-        qCDebug(scriptengine) << "ScriptEngine::loadEntityScript() entity:" << entityID << "\n"
-            "    NOT CONSTRUCTOR\n"
-            "    SCRIPT:" << scriptOrURL;
+        QString testConstructorType = QString(testConstructor.toVariant().typeName());
+        if (testConstructorType == "") {
+            testConstructorType = "empty";
+        }
+        QString testConstructorValue = testConstructor.toString();
+        if (testConstructorValue.size() > 80) {
```
Comment (ZappoMan) [inline libraries/script-engine/src/ScriptEngine.cpp]: note: magic numbers.... and you even typed it twice... tsk tsk

**Comment by hifi-gustavo** [comment]
Build console output for commit: 672b4df7c13b0e0a9d42b38bd8fa9c2510989f57 available at: https://gist.github.com/4ffafd84154a88eb3ccca2b3128c7378<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7600.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7600.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 672b4df7c13b0e0a9d42b38bd8fa9c2510989f57 available at: https://gist.github.com/4ffafd84154a88eb3ccca2b3128c7378<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7600.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7600.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 7 - Additional log output from this PR will be of use.  It's *much* better than current production build output. :+1:

**Pull request merged**